.github/workflows/release.yaml

```diff
@@ -311,8 +311,10 @@ jobs:
         run: |
           make manifests/crd/release CHART_VERSION="${VERSION_WITHOUT_PREFIX}"
 
-          make manifests/kubernetes IMAGE="docker.io/dynatrace/dynatrace-operator" TAG="${VERSION}"
-          make manifests/openshift IMAGE="registry.connect.redhat.com/dynatrace/dynatrace-operator" TAG="${VERSION}"
+          make manifests/kubernetes/olm IMAGE="public.ecr.aws/dynatrace/dynatrace-operator" TAG="${VERSION}"
```

Comment:
```
for Item in "manifests/kubernetes/olm" "manifests/kubernetes" "manifests/openshift";
  do
    make $Item IMAGE="public.ecr.aws/dynatrace/dynatrace-operator" TAG="${VERSION}"
  done
  make manifests/openshift/olm IMAGE="registry.connect.redhat.com/dynatrace/dynatrace-operator" TAG="${VERSION}"
```

Reply:
=)

Reply:
not sure that these 3 lines better then previous, so ignore it and good to go :)